src/Styles/InputTime.scss

```diff
@@ -34,5 +34,10 @@
         color: black;
       }
     }
+
+    &-up {
+      margin-bottom: -5px;
+      margin-top: 2px;
```

Comment:
i think it is more suitable with no margin-top & set margin-bottom to -3px, isn't it ?

Reply:
I couldn't get quite the same alignment with just margin-bottom: -3px.

The margin-top forces them both down a bit (otherwise they felt a bit uncentered) and the margin-bottom helped bring them closer together.

There might be a nicer one-liner that could achieve the same though?

Reply:
hmm, haven't thought about it. Let's leave it this way